src/providers/completer/glossary.ts

```diff
@@ -39,7 +39,7 @@ export class Glossary implements IProvider {
         if (result[1] && result[1].match(/^ac/i)) {
             suggestions = this.acronyms
         } else {
-            suggestions = Object.assign({}, this.acronyms, this.glossaries)
+            suggestions = new Map( [...this.acronyms.entries(), ...this.glossaries.entries()] )
```

Comment:
According to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map#cloning_and_merging_maps, we can directly write

```js
suggestions = new Map( [...this.acronyms, ...this.glossaries] )
```